examples/render/contour.yaml

```diff
@@ -209,7 +72,965 @@ data:
+                            unhealthyThresholdCount:
+                              description: The number of unhealthy health checks required
+                                before a host is marked unhealthy
+                              format: int32
+                              type: integer
+                          required:
+                          - path
+                          type: object
+                        name:
+                          description: Name is the name of Kubernetes service to proxy
+                            traffic. Names defined here will be used to look up corresponding
+                            endpoints which contain the ips to route.
+                          type: string
+                        port:
+                          description: Port (defined as Integer) to proxy traffic
+                            to since a service can have multiple defined
+                          type: integer
+                        strategy:
+                          description: LB Algorithm to apply (see https://github.com/projectcontour/contour/blob/master/design/ingressroute-design.md#load-balancing)
+                          type: string
+                        validation:
+                          description: UpstreamValidation defines how to verify the
+                            backend service's certificate
+                          properties:
+                            caSecret:
+                              description: Name of the Kubernetes secret be used to
+                                validate the certificate presented by the backend
+                              type: string
+                            subjectName:
+                              description: Key which is expected to be present in
+                                the 'subjectAltName' of the presented certificate
+                              type: string
+                          required:
+                          - caSecret
+                          - subjectName
+                          type: object
+                        weight:
+                          description: Weight defines percentage of traffic to balance
+                            traffic
+                          format: int32
+                          type: integer
+                      required:
+                      - name
+                      - port
+                      type: object
+                    type: array
+                  timeoutPolicy:
+                    description: The timeout policy for this route
+                    properties:
+                      request:
+                        description: Timeout for receiving a response from the server
+                          after processing a request from client. If not supplied
+                          the timeout duration is undefined.
+                        type: string
+                    required:
+                    - request
```

Comment:
This one is questionable, but I think for the case of IngressRoute which we don't want to change, this field should be optional even if not supplying it makes no sense.